kubeflow-katib.yaml

```diff
@@ -31,7 +30,7 @@ pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/kubeflow/katib
-      expected-commit: 05ab6f012ab6723609700a47a6cfa1845080e9b6
+      expected-commit: fc6beec8358f1c4a50d86de63c5906fce2eba373
       tag: v${{package.version}}
 
   - uses: patch
```

Comment:
Fly-by comment! In the subpackage section for `name: "katib-${{range.key}}"`, could you please drop the `deps` for `github.com/docker/distribution@v2.8.2 github.com/docker/docker@v20.10.24`, and add `google.golang.org/grpc@v1.56.3`? It appears that upstream has upgraded their docker deps.

This will fix the CVE check!